Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 # PostgreSQL driver
-gem 'pg'
+gem 'pg', '0.20'
```

Comment:
ah, shoulda left a comment on it. postgres kept displaying a deprecation warning, something we kept seeing last friday and when I cloned I started getting on my own machine. This just locks postgres to a specific version so that there isn't a version conflict. the latest version, 0.21 is what we were running before